packages/principles-core/src/runtime-v2/adapter/__tests__/openclaw-cli-runtime-adapter-message-file.test.ts

```
@@ -0,0 +1,225 @@
+    });
+
+    it('deletes the message file after a TIMED-OUT probe', async () => {
+      const adapter = new OpenClawCliRuntimeAdapter({
+        runtimeMode: 'local',
+        agentId: 'diag',
+        workspaceDir: tempWorkspace,
+      });
+      stubFirstTwoProbes();
+      // Capture the file path before the probe runs so we can check cleanup
+      // after the early-return on timeout.
+      let createdFilePath = '';
```

ESLint

[error] The value assigned to 'createdFilePath' is not used in subsequent statements.

(no-useless-assignment)